mscore/editstyle.ui

```diff
@@ -10606,7 +10606,6 @@
   <tabstop>evenFooterR</tabstop>
   <tabstop>showMeasureNumber</tabstop>
   <tabstop>showFirstMeasureNumber</tabstop>
-  <tabstop>showAllStaffsMeasureNumber</tabstop>
```

Comment:
This deletion should not be present. So please edit tabstops too if there're widget changes.

Reply:
It has been moved. It's still there, but as showAll**Staves**MeasureNumber

Reply:
It's recommended to edit tabstops immediately after editing widgets in Qt Creator/Designer.

Reply:
Hum. No. You are right. It has been removed.